collections/VideosCollection.js

```diff
@@ -0,0 +1,234 @@
+var VideosCollection = Backbone.Collection.extend({
+
+  url: `https://youtube.googleapis.com/youtube/v3/search?part=snippet&`,
+  
+  model: VideoModel,
+
+  addVideo: function (title, description, id) {
+    this.add({
+      title: title,
+      description: description,
+      id: id
+    });
+  },
+
+
+  getResults: function (query) {
+    var search_params = {
+      'key': 'AIzaSyBW1oAZn9yrQDhfoZPkwnHvVNDUsJRe7Ho',
+      'q': query,
+      'type': 'video',
+      'limit': 5
+    };
+
+    return this.fetch(
+      { 
+        data: $.param(search_params),
+        reset: true
+      });
+  },
+
+// Response is parsed after every fetch event.  
+  parse: function (response) {
+    return response.items.map(function (v) {
+      if (v.id.kind == 'youtube#video') {
+        var newObj = {
+          title: v.snippet.title,
+          description: v.snippet.description,
+          id: v.id.videoId,
+          link: `https://www.youtube.com/watch?v=${v.id.videoId}`,
+          thumbnail_link: v.snippet.thumbnails.default.url
+        }
+
+        return Object.assign({}, v, newObj);
+
+      } else {
+        return null;
+      }    
+    }
+  )}
+});
+
+
+var sampleData = {
+  "kind": "youtube#searchListResponse",
+  "etag": "jrLgj0XGHegO7nztkfHkqGu_dmo",
+  "nextPageToken": "CAUQAA",
+  "regionCode": "US",
+  "pageInfo": {
```

Comment:
Having mock data is fine for local development but hurts readability a bit when making a PR